lib/Runtime/Library/JavascriptBigInt.cpp

```diff
@@ -397,4 +414,183 @@ namespace Js
         return (leftBigInt->Compare(rightBigInt) == 0);
     }
 
+    // pbi1 += pbi2 assume pbi1 > pbi2
```

Comment:
If there's an assumption on the input data, it's nice to add it as an Assert within the top of the method.

Reply:
should we add it if that checking is expensive?

Reply:
Yeah, `Assert` is only included in debug builds so it's fine to do expensive stuff there. In test or release builds, it goes away.

---
In reply to: [227876426](https://github.com/Microsoft/ChakraCore/pull/5792#discussion_r227876426) [](ancestors = 227876426)